server/controllers/get-contract-by-id.js

```diff
@@ -38,8 +38,9 @@ module.exports = exports = async (req, res, next) => {
 			if (req.query.save !== '0') {
 				let contract_data = reformatSalesforceContract(JSON.parse(JSON.stringify(contract)));
 				contract_data.last_updated = new Date();
-				if (contract_data.orders) {
-					const activeOrder = contract_data.orders.find(order => order.status === 'Active');
+				if (contract.orders) {
+					const now = new Date();
+					const activeOrder = contract.orders.find(order => order.status === 'Activated' && new Date(order.startDate) <= now && new Date(order.endDate) >= now);
```

Comment:
**Question**: Can there only be one activated order at time?

Reply:
@asugar13 Salesforce team have made changes on their side to filter out orders that are not referencing the current active year, ideally it would only be one. The timeframe check is for extra validation since it could be that some amendments get missed by the filter, and they could also have the 'Activated' status.